frontend/ui/elements/menu_keyboard_layout.lua

```diff
@@ -66,18 +69,13 @@ local sub_item_table = {
         separator = true,
     },
     {
-        text_func = function()
-            return T(_("Keyboard settings: %1pt%2%3%4"),
-                G_reader_settings:readSetting("keyboard_key_font_size", VirtualKeyboard.default_label_size),
-                G_reader_settings:isTrue("keyboard_key_bold") and string.format(", %s", _("bold")) or "",
-                G_reader_settings:nilOrTrue("keyboard_key_border") and string.format(", %s", _("border")) or "",
-                G_reader_settings:isTrue("keyboard_key_compact") and string.format(", %s", _("compact")) or "")
-        end,
+        text = _("Keyboard settings"),
         keep_menu_open = true,
         callback = function(touchmenu_instance)
             input_dialog = require("ui/widget/inputdialog"):new{
                 title = _("Keyboard font size"),
-                input = tostring(G_reader_settings:readSetting("keyboard_key_font_size", VirtualKeyboard.default_label_size)),
+                input_type = "number",
```

Comment:
Please do not switch to the numeric layer bacause:
-while setting the keyboard it is convenient to see the letters and how they look
-we have numeric keys in the upper row